### PR TITLE
add basic html and css of Loki

### PR DESCRIPTION
**Pull request by luligodio**

### CODE REVIEW

**Review by lyondeveloper — APPROVED** [review]
me gusto la tabla que agregaste! LGTM

**Comment by Itnardoel** [comment]
El gif de "Get Help" es lo mas

File: Luciana Godio/index.html [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+        <img class="loki-verde" src="images/Loki-verde.jpg" alt="loki-verdoso">
+        <p>This page is dedicated to the character <strong>Loki</strong> from the Marvel Cinematic Universe.</p>
+        <p>Loki Laufeyson is the God of Mischied, Evil and Lies. Is also know as the Thickster God. </p>
+        <p>He was the biological son of Laufey, King of the Frost Giants, who was abandoned and left to die shortly
+            after his birth.
+            Found by Odin, was taken to Asgard and raised by him and Frigga as an Asgardian prince, along with Thor.</p>
+        <p>Al recorrer el multiverso de marvel, se ha conocido que hay diferentes versiones de Loki en cada universo.
+            Loki es desde un humano hasta un cocodrilo.</p>
+        <div class="container">
+            <div class="quote1">
```
Comment (rominasteinberg) [inline Luciana Godio/index.html:45]: Check the q tag if it suits for your case

File: Luciana Godio/index.html [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+            <tr>
+                <td>Adoptive Brother</td>
+                <td>Thor</td>
+            </tr>
+        </table>
+    </section>
+    <section id="powers">
+        <h3>Powers</h3>
+        <p>Some of Loki's powers are:</p>
+        <div class="contenedor-lista">
```
Comment (rominasteinberg) [inline Luciana Godio/index.html:91]: lets use only one language to name classes or ids

File: Luciana Godio/style.css [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+    align-items: center;
+    background-color: black;
+    width: 100vw;
+    height: 60px;
+    margin: -9px 0 0 -9px;
+    position: sticky;
+    z-index: 1;
+    
+}
+.nav-lista{
```
Comment (rominasteinberg) [inline Luciana Godio/style.css:12]: suggestion: order by tags, classes and ids in this order to make easier to read the file

File: Luciana Godio/style.css [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+    /* border: 4px solid green; */
+    text-align: center;
+    margin:auto;
+}
+.container{
+    display: grid;
+    grid-template-columns: auto auto;
+    grid-template-rows: auto auto;
+    gap: 6px 6px;
+    grid-template-areas:
```
Comment (rominasteinberg) [inline Luciana Godio/style.css:64]: great job using grid-template-areas

File: Luciana Godio/style.css [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+    grid-area: ldagger;
+}
+.image-dagger {
+    grid-area: image-dagger;
+}
+.quote2 {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 0.1fr 1fr;
+    gap: 0px 0px;
```
Comment (rominasteinberg) [inline Luciana Godio/style.css:88]: perhaps you can group diplay, grid-template-column, grid-template-rows and gap in another class, and just leave here what is proper for quote2. and the rest create a separated class that can be reused

**Pull request merged**